lib/Gocdb_Services/Role.php

```diff
@@ -927,7 +973,8 @@ private function selfGrantRole(\User $primaryUser, \User $currentUser, \Role $ro
      * @param string $ownedEntityType One of ngi, site, service, project, servicegroup
      * @return array of {@link \RoleActionRecord}s
      */
-    public function getRoleActionRecordsById_Type($id, $ownedEntityType){
+    public function getRoleActionRecordsById_Type($id, $ownedEntityType)
```

Comment:
Why not rename this to `getRoleActionRecordsByIdType`?

Reply:
Because that would add several more code files to the PR which was only supposed to 'fix' those touched by the original PR. That might just be laziness, but then do we standardise view_project, view_sgroup ...  and where does that end? Compromise would be just to fix the naming.